--- origsrc/gcc-4.7.3/libstdc++-v3/configure	2012-11-05 17:42:32.000000000 -0600
+++ src/gcc-4.7.3/libstdc++-v3/configure	2013-05-12 12:29:39.573224400 -0500
@@ -45520,7 +45520,7 @@ $as_echo "#define HAVE_TLS 1" >>confdefs
 	;;
     esac
     ;;
-  *-linux* | *-uclinux* | *-gnu* | *-kfreebsd*-gnu | *-knetbsd*-gnu)
+  *-linux* | *-uclinux* | *-gnu* | *-kfreebsd*-gnu | *-knetbsd*-gnu | *-cygwin*)
 
   # All these tests are for C++; save the language and the compiler flags.
   # The CXXFLAGS thing is suspicious, but based on similar bits previously
--- origsrc/gcc-4.7.3/libstdc++-v3/crossconfig.m4	2011-07-18 06:04:18.000000000 -0500
+++ src/gcc-4.7.3/libstdc++-v3/crossconfig.m4	2013-05-12 12:27:20.983033200 -0500
@@ -141,7 +141,7 @@ case "${host}" in
 	;;
     esac
     ;;
-  *-linux* | *-uclinux* | *-gnu* | *-kfreebsd*-gnu | *-knetbsd*-gnu)
+  *-linux* | *-uclinux* | *-gnu* | *-kfreebsd*-gnu | *-knetbsd*-gnu | *-cygwin*)
     GLIBCXX_CHECK_COMPILER_FEATURES
     GLIBCXX_CHECK_LINKER_FEATURES
     GLIBCXX_CHECK_MATH_SUPPORT
